programs-remote-listings/rs-connect-shortcodes.php

```diff
@@ -22,18 +22,19 @@ public function shortcode_programs($atts)
         $vars = 'fields=_program_base_,_program_registration_';
 
         if (empty($shortcode_atts['table'])) {
+            $vars .= ',price_details';
```

Comment:
I wonder if there's any advantage to excluding/including stuff in the payload. Why not just have $vars = to all of the fields needed... just a thought